cime_config/tests.py

```diff
@@ -627,15 +627,15 @@
     "e3sm_scream_v1_dp-eamxx" : {
         "time"  : "01:00:00",
         "tests" : (
-            "ERS_Ln22.ne30_ne30.F2010-SCREAMv1-DP-DYCOMSrf01", # 225 phys cols, roughly size of ne2
+            "ERS_Ln22.ne30_ne30.F2010-SCREAMv1-DP-DYCOMSrf01.scream-ntasks-16", # 225 phys cols, roughly size of ne2
```

Comment:
If you're going to hardcode the test to 16 tasks, you don't need a testmod. Just do:
`ERS_P16_Ln22.ne30_ne30.F2010-SCREAMv1-DP-DYCOMSrf01`

Reply:
ahh good to know! Thanks.

Reply:
Ah, just read this after the prev comment. I was going to suggest the same thing.